cli/actions/jwt_secrets_test.go

```diff
@@ -0,0 +1,57 @@
+package actions_test
+
+import (
+	"github.com/NethermindEth/sedge/cli/actions"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"testing"
+)
+
+func TestSedgeActions_CreateJwtSecretsContainers(t *testing.T) {
+	tests := []struct {
+		name    string
+		options actions.CreateJwtSecretOptions
+		err     bool
+	}{
+		{
+			name: "Wrong Jwt Path",
+			options: actions.CreateJwtSecretOptions{
+				GenerationPath: t.TempDir(),
+				JwtPath:        filepath.Join("a", "b", "c"),
+				Network:        "mainnet",
+			},
+		},
+		{
+			name: "Missing network",
+			options: actions.CreateJwtSecretOptions{
+				GenerationPath: t.TempDir(),
+				JwtPath:        "",
+			},
+			err: true,
+		},
+		{
+			name: "Generation of jwt",
+			options: actions.CreateJwtSecretOptions{
+				GenerationPath: t.TempDir(),
+				JwtPath:        "",
+				Network:        "mainnet",
+			},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			sedgeActions := actions.NewSedgeActions(nil, nil, nil)
+			jwtPath, err := sedgeActions.CreateJwtSecrets(tc.options)
+			if err != nil {
+				if !tc.err {
+					t.Errorf("Got: %v, want error: %v", err, tc.err)
+				}
+				return
+			}
+			if tc.err || tc.options.JwtPath != "" {
+				return
```

Comment:
I believe that if `tc.error` is true and `err` is nil then the test failed. If I am correct then please replace the `return` with a `t.Error(...)` and move `tc.options.JwtPath != ""` to another `if` statement

Reply:
You were right, done at [e83617d](https://github.com/NethermindEth/sedge/pull/210/commits/e83617d10d7504004f476ab31542064e7d143786)